run-cf-tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+#set -x
+set -o pipefail
+
+### This script runs the Checker Framework's tests that depend on
+### dljc (this project). It is intended to be used as part of a CI integration test.
+### This script must be run from the top-level directory
+### It performs these actions:
```

Comment:
Nit: I'm not sure this comment does much good.  It could be inlined into the section below.